airflow/docker_image.go

```diff
@@ -276,6 +275,92 @@ func (d *DockerImage) TagLocalImage(localImage string) error {
 	return nil
 }
 
+func (d *DockerImage) RunTest(dagID, envFile, settingsFile, startDate, containerName string, taskLogs bool) error {
+	// delete container
+	stdout := os.Stdout
+	stderr := os.Stderr
+	err := cmdExec(DockerCmd, nil, nil, "rm", "astro-run")
+	if err != nil {
+		log.Debug(err)
+	}
+	var args []string
+	if containerName != "" {
+		args = []string{
+			"exec",
+			"-t",
+			containerName,
+		}
+	}
+	// docker exec
+	if containerName == "" {
+		// convert Settings file to variables and connection yaml
+		err = settings.FileToConnectionYAML(settingsFile)
+		if err != nil {
+			log.Debug(err)
+		}
+		args = []string{
```

Comment:
```suggestion
		}
		defer func() {
			// delete files
	                os.Remove("./variables.yaml")
	                os.Remove("./connections.yaml")
		}()
		args = []string{
```
always better to defer & delete right after the creation, instead of adding it at the end of the function

Reply:
okay